### PR TITLE
Add Aaron Network

### PR DESCRIPTION
**Pull request by aaronetwork**

### CODE REVIEW

File: aaronetwork/chain.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "aaronetwork",
```
Comment (JeremyParish69) [inline aaronetwork/chain.json:3]: I think this chain_name will lead to a lot of confusion. If keeping the 'network' part, then I'd suggest "aaronnetwork" instead.

File: aaronetwork/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      },
+      "socials": {
+        "website": "https://aaronetwork.xyz",
+        "twitter": "https://x.com/aaronetwork"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/aaronetwork/images/aaron.png",
+          "theme": {
+            "circle": true
```
Comment (JeremyParish69) [inline aaronetwork/assetlist.json:33]: this image is not circular. Please change to false

File: aaronetwork/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply aaronetwork/chain.json:3]: Btw, this suggestion is not a requirement. I see that the merging of the two 'n's is used in many places (chain_id, website, twitter, explorer url, etc.)

Comment (aaronetwork) [reply aaronetwork/chain.json:3]: Should chain_name be fused? Can we specify "aaron network" or only "aaronnetwork"?

Comment (JeremyParish69) [reply aaronetwork/chain.json:3]: Space characters are not allowed. This is just for identifying, and never shown to app users.

**Pull request merged**